package.json

```diff
@@ -71,6 +71,19 @@
     "typescript": "^3.3.3",
     "ws": "^6.2.0"
   },
+  "files": [
```

Comment:
The `files` section is an inclusion list, so this should be the list of files/folders to **include**, not **exclude**.

Reply:
Ohh, Sorry @KarishmaGhiya accidentally gave me the wrong list. I'll update it.

Reply:
I think the following list might be exactly what we want, but I'm not 100% sure:

```
"files": [
  "browser/",
  "dist/",
  "dist-esm/lib/",
  "lib/",
  "typings/lib"
]
```

The other files like LICENSE, README, and CHANGELOG are auto-included by npm so I would leave them out of package.json for simplicity.

@bterlson: Do you agree with the above list?

Reply:
Looks good to me!